.ci_helpers/docker/download.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# ==== ONLY EDIT WITHIN THIS BLOCK =====
+DATA_PATH=/usr/local/apache2/htdocs/data
+
+ek60_source='https://ncei-wcsd-archive.s3-us-west-2.amazonaws.com/data/raw/Bell_M._Shimada/SH1707/EK60/Summer2017-D20170615-T190214.raw'
```

Comment:
@emiliom Could you go over these datasets and let me know if we need more? Thanks!

Reply:
Heh, I don't think I'm the right person for that; @leewujung is. As far as I'm concerned, you've included one file of each instrument type (except ea640, and I don't know anything about that), so what you have looks good.